src/screens/MainApp/Home/Dashboard/DiagnosesBoard/BoardItem/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { View, Text, Image } from 'react-native'
+import styles from './styles'
+import cannabisLeaf from './resources/placeholder.png'
+import answeredIcon from './resources/answered.png'
+import notAnsweredIcon from './resources/notAnswered.png'
+
+const RealThumbnailOrPlaceholder = ({ thumbnail }) => (
+  <Image
+    source={thumbnail ? { uri: thumbnail } : cannabisLeaf}
+    style={styles.thumbnail}
+  />
+)
+
+const AnsweredIcon = ({ answered }) => {
+  const icon = answered ? answeredIcon : notAnsweredIcon
+  const style = answered ? styles.answered : styles.notAnswered
+  return <Image source={icon} style={style} />
+}
+
+const BoardItem = ({ thumbnail, diagnose }) => {
+  const moment = require('moment')
```

Comment:
Please consider moving `require` to the top. Also, we use `import`.
Something like this:
`import { View, Text, Image } from 'react-native'`
`import moment from 'moment'`